src/commons/sideContent/SideContentContestVoting.tsx

```diff
@@ -17,79 +17,190 @@ type StateProps = {
   const contestEntryCards = useMemo(
     () => (
-      <div>
-        {contestEntryHeader}
-        {contestEntries.length > 0 ? (
-          contestEntries.map((contestEntry: ContestEntry, index) => (
-            <SideContentContestEntryCard
-              isValid={isValid}
-              canSave={canSave}
-              entryNumber={index + 1}
-              key={contestEntry.submission_id}
-              handleContestEntryClick={handleContestEntryClick}
-              handleVotingSubmissionChange={handleVotingSubmissionChange}
-              contestEntry={contestEntry}
-              minScore={minScore}
-              maxScore={maxScore}
-            />
-          ))
+      <div className="tier-list">
+        {tierBoard}
+        {sortedContestEntries.length > 0 ? (
+          <div className="tier" id="bank">
+            <div
+              className="item-container"
+              id="items"
+              onDragOver={handleDragOver}
+              onDragLeave={handleDragLeave}
+              onDragEnter={handleDragEnter}
+              onDrop={handleDrop}
+            >
+              {sortedContestEntries.map((contestEntry: ContestEntry, index) => (
+                <div
+                  className={classNames('item', { wrong: !isValid })}
+                  draggable={canSave}
+                  key={`item-${index + 1}`}
+                  id={`item-${index + 1}`}
+                  onDragStart={handleDragStart}
+                  onDragEnd={handleDragEnd(contestEntry)}
+                  ref={item => (contestEntryRefs.current[index] = item)}
+                >
+                  <Card
+                    className={Classes.INTERACTIVE}
+                    elevation={Elevation.ONE}
+                    onClick={() =>
+                      handleContestEntryClick(
+                        contestEntry.submission_id,
+                        contestEntry.answer.code ?? ''
+                      )
+                    }
+                  >
+                    <Pre className="contestentry-entryid">{index + 1}</Pre>
+                  </Card>
+                </div>
+              ))}
+            </div>
+          </div>
         ) : (
           <div className="noResults">There are no eligible entries for voting found.</div>
         )}
       </div>
     ),
-    // determines when to re-render
     [
       isValid,
       canSave,
-      contestEntries,
+      sortedContestEntries,
       handleContestEntryClick,
-      handleVotingSubmissionChange,
-      maxScore,
-      minScore
+      handleDragEnd,
+      handleDragEnter,
+      handleDragLeave,
+      handleDragOver,
+      handleDrop,
+      tierBoard
     ]
   );
 
+  // Upon initial render, loads contest entries in the correct tier using saved score
+  useEffect(() => {
+    sortedContestEntries.forEach((entry, index) => {
+      if (entry.score !== null) {
+        const savedTierIndex = TIERS.findIndex(tier =>
+          entry.score ? tier.score <= entry.score : false
```

Comment:
```suggestion
          // We are using <= instead of === backwards
          // compatibiilty with legacy voting submisions.
          entry.score ? tier.score <= entry.score : false
```